### ISSUE TITLE
Add "Scroll to Top" Button for Improved Navigation and User Experience

### ISSUE THREAD

**Issue by piyushshrma** [PriyaGhosal/SkillWise#19]
To enhance the user experience and improve overall site navigation on SkillWise, I propose the addition of a "Scroll to Top" button. This feature would allow users to quickly return to the top of the page without needing to manually scroll, especially on pages with long content. By reducing the need for excessive scrolling, this feature will streamline the browsing experience, making the platform more user-friendly and efficient. Implementing this functionality will help prevent frustration for users, particularly on mobile devices or lengthy pages.

Kindly assign me this issue under Hacktoberfest 2024.

**Comment by PriyaGhosal** [comment]
Assigned!
Make sure to attach before & after screenshots in your PR. Also, star the repo before working on it. While creating your PR, ensure you pull all existing changes beforehand to avoid conflicts.
Happy open-source contributing!

**Comment by swamimalode07** [comment]
![Screenshot 2024-09-29 174316](https://github.com/user-attachments/assets/50862d73-e007-48f7-bcd5-725ad8b40f81)

Added a Scroll to top button , should i make a pull request

**Comment by swamimalode07** [comment]
Made this button according to your User interface with hover effect

**Comment by swamimalode07** [comment]
I have made a pull request , kindly merge it

**Comment by piyushshrma** [comment]
Contribution: Added "Scroll to Top" Feature in Skillwise
In my recent contribution to the Skillwise open-source project, I implemented a "Scroll to Top" button to improve the overall user experience. This feature allows users to quickly navigate back to the top of the page, especially after scrolling through long content sections.

Feature Added: Scroll to Top button
Location: Bottom-right corner of the webpage, visible after scrolling down
Improvement: Enhanced user experience by providing a seamless and convenient way to return to the top of the page without manual scrolling, making navigation smoother.
This feature makes the site more user-friendly, especially for content-heavy pages, and was appreciated by the maintainers for improving usability.

![Screenshot 2024-09-29 232215](https://github.com/user-attachments/assets/20a8fd10-a38a-43b6-b10c-7a0763016d56)
